test/lib/agent_helper.js

```diff
@@ -71,639 +65,572 @@ function FakeSegment(transaction, duration, name = 'FakeSegment') {
   }
 }
 
-const helper = (module.exports = {
```

Comment:
I really like that this sort of declaration has been removed. It's quite difficult to read and keep track of. I think the refactored declaration is also going to help quite a lot with jsdoc and ide integration. The only thing missing from the refactor is the removal of the singleton `_agent`.

Reply:
yea this was something I wasn't planning on doing but had to edit a few things here so broke this down.  The agent singleton is going to be a heavier lift. I don't know the effects of this.  I can log a story in our cleanup feature for this